hilfe/handbuch/mitarbeiter/dokumente/index.md

```diff
@@ -71,6 +73,79 @@ in die Dokumentenübersicht zum gewünschten Mitarbeiter. Anschließend
 > Verzeichnis **Dieser PC/Downloads** gespeichert und kann von hier in Pentacode
 > importiert werden.
 
+## Personalfragebogen erstellen
+
+Pentacode erlaubt Ihnen auch vorgefertigte Dokumente vom Mitarbeiter von
+Mitarbeitern ausfüllen zu lassen. Dies ist besonders hilfreich um schnell
+Daten und Informationen einzuholen und direkt in Pentacode zu speichern. Zur
+Zeit bietet Pentacode ihnen den **Personalfragebogen**, welchen Sie wie folgend
+anlegen können:
+1. Wählen Sie den gewünschten Mitarbeiter aus und gehen zu **Dokumente & Formulare**.
+2. Klicken Sie auf das {%icon "file-circle-plus"%} **-Symbol** in der oberen
+   rechten Ecke.
+3. Wählen Sie {%icon "pen-field"%} **Personalfragebogen** aus. 
+4. Fügen Sie einen Kommentar und eine Kategorie hinzu, falls gewünscht und
+   klicken Sie auf **Speichern**
+
+Der vorläufige Personalfragebogen ist nun im Dokumentenarchiv des Mitarbeiters
+gespeichert und noch mit dem Zusatz 'Entwurf' versehen. Sie müssen ihn erst
+[ausfüllen und freigeben](#personalfragebogen-ausfüllen-und-freigeben) bevor der
+Mitarbeiter an der Reihe ist.
+
+
+>**Tipp**: {%icon "lightbulb"%} Versehen Sie den Personalfragebogen mit einer
+>Kategorie, die Sie am besten auch 'Personalfragebogen' nennen. So finden Sie
+>auch unter vielen Dokumenten immer schnell das richtige. 
+
+### Doch warum ein Personalfragebogen?
+
+Das Übertragen der **Sozialversicherungs- und Lohndaten** neuer Mitarbeiter
+nimmt besonders in Betrieben mit häufig wechselnder Belegschaft viel Zeit in
+Anspruch. Mit Hilfe von **Personalfragebögen** können Sie neuen Mitarbeitern ein
+Formular zuschicken, in welchem sie ihre Daten angeben können und diese direkt
+und sicher in ihrem Pentacode Dokumentenarchiv gespeichert werden. So sparen Sie
+sich Zeit die Lohndaten einzufordern und müssen sie auch nicht mehr händisch
+übertragen, was Fehler vermeidet.
+Wenn Sie nun noch die
+[Sofortmeldungsfunktion](/hilfe/handbuch/mitarbeiter/allgemein/) nutzen,
+werden die Daten automatisch an die Rentenversicherung weitergeleitet ohne, 
+dass Sie auch nur einmal Sozialversicherungsnummer und co. händisch eintragen
+müssen. 
+
+### Personalfragebogen ausfüllen und freigeben
+
+Sobald Sie nun auf den gerade erstellten Personalfragebogen klicken, der unter
+den Dokumenten des Mitarbeiters gespeichtert ist, öffnet sich dieser und steht
+Ihnen zum Ausfüllen bereit. Tragen Sie nun die nötigen Informationen zu ihrem
+Betrieb, dem Vertragsverhältnis mit dem Mitarbeiter, sowie Vergütung und
+Arbeitszeit ein und klicken Sie auf {%icon "paper-plane"%}. Der Mitarbeiter wird nun informiert und kann den
```

Comment:
Bitte für buttons auch wirklich immer `{% button %}` inclusive label verwenden und optimalerweise auch mit dem richtigen styling. In diesem fall:
```njk
{% button icon="paper-plane", label="Zum Ausfüllen Freigeben" %}
```
```